Sources/ArcGISToolkit/Extensions/ArcGIS/Basemap.swift

```diff
@@ -0,0 +1,24 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+
+extension Basemap {
+    /// A Boolean value indicating whether the basemap supports 3D visualization.
+    var is3D: Bool {
+        baseLayers.contains {
+            $0 is ArcGISSceneLayer
+        }
```

Comment:
We typically don't use a trailing closure when calling a method with a single unmuted closure parameter.

```suggestion
        baseLayers.contains(where: { $0 is ArcGISSceneLayer })
```